include/uapi/sound/sof/tokens.h

```diff
@@ -96,7 +96,7 @@
  */
 #define SOF_TKN_COMP_SINK_PIN_BINDING_WNAME	413
 #define SOF_TKN_COMP_SRC_PIN_BINDING_WNAME	414
-
+#define SOF_TKN_COMP_PAYLOAD_WITH_OUTPUT_FMT	415
```

Comment:
you have a SOF PR for this?

Reply:
the token names in the commit message and code do notmatch

Reply:
and im asking for a simple SOF PR that adds this token only in tokens.conf. You dont need any module for that do you?

Reply:
@ranj063 Got it. SOF PR is submitted: https://github.com/thesofproject/sof/pull/6500.

> the token names in the commit message and code do notmatch

Thanks. Will update it.